<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualSearch</class>
 <widget class="QWidget" name="ManualSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*＊✿❀  Manual Search  ❀✿＊*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="species">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Species</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="dogCheckBox">
      <property name="text">
       <string>Dog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="catCheckBox">
      <property name="text">
       <string>Cat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rabbitCheckBox">
      <property name="text">
       <string>Rabbit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rodentCheckBox">
      <property name="text">
       <string>Rodent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="fishCheckBox">
      <property name="text">
       <string>Fish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="birdCheckBox">
      <property name="text">
       <string>Bird</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="age">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Age</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="youngCheckBox">
      <property name="text">
       <string>Young</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="adultCheckBox">
      <property name="text">
       <string>Adult</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="seniorCheckBox">
      <property name="text">
       <string>Senior</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="maleCheckBox">
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="femaleCheckBox">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="size">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Size</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QCheckBox" name="smallCheckBox">
      <property name="text">
       <string>Small</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mediumCheckBox">
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="largeCheckBox">
      <property name="text">
       <string>Large</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="temperament">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Temperament</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QCheckBox" name="happyCheckBox">
      <property name="text">
       <string>Happy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="friendlyCheckBox">
      <property name="text">
       <string>Friendly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="introvertedCheckBox">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Introverted</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="activeCheckBox">
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="vigilantCheckBox">
      <property name="text">
       <string>Vigilant</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="goodWith">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Good With</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QCheckBox" name="kidsCheckBox">
      <property name="text">
       <string>Kids</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="animalsCheckBox">
      <property name="text">
       <string>Animals</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="shelter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Shelter</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="bestFriendsCheckBox">
      <property name="text">
       <string>Best Friends</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="humaneSocietyCheckBox">
      <property name="text">
       <string>Humane Society</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="animalWelfareLeagueCheckBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Animal Welfare League</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>600</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QLabel" name="pic2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>Pet pic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="name2">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="link2">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(161, 212, 255)</string>
        </property>
        <property name="text">
         <string>Learn More</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="link3">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: rgb(161, 212, 255)</string>
        </property>
        <property name="text">
         <string>Learn More</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="link1">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(161, 212, 255)</string>
        </property>
        <property name="text">
         <string>Learn More</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="name3">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pic1">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>Pet pic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="name1">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="pic3">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>Pet pic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="pageLine">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Showing page # out of #</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="previous">
        <property name="text">
         <string>← Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next">
        <property name="text">
         <string>Next →</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="searchBar"/>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
